src/bindings/python/src/openvino/utils.py

```diff
@@ -18,7 +18,11 @@ def add_openvino_libs_to_path() -> None:
         # add the location of openvino dlls to your system PATH.
         #
         # looking for the libs in the pip installation path by default.
-        openvino_libs = [os.path.join(os.path.dirname(__file__), "libs")]
+        openvino_libs = [os.path.join(os.path.dirname(__file__), "libs"),  # find libs for pip installation
+                         os.path.join(os.path.dirname(__file__), "..", "..", ".."),  # find libs in bin directory
+                         os.path.join(os.path.dirname(__file__), "..", "..", "..", "..", "..", "..", "temp", "tbb", "bin"),  # find tbb libs
+                         os.path.join(os.path.dirname(__file__), "..", "..", "..", "runtime", "bin", "intel64", "Release"),  # find libs for installed pkg
+                         os.path.join(os.path.dirname(__file__), "..", "..", "..", "runtime", "3rdparty", "tbb", "bin")]  # find tbb libs for installed pkg
```

Comment:
not sure that it's secure to probe great amount of locations.

Reply:
I can exclude the two last paths as it's covered by setupvars.bat script

Reply:
done